agent/security_helper.py

```diff
@@ -118,3 +120,74 @@ def selinux_status():
             selinux_mode = row[1].strip()
 
     return {'enabled': selinux_enabled, 'mode': selinux_mode}
+
+
+AUDITED_CONFIG_FILES = [
+    '/etc/passwd',
+    '/etc/shadow',
+    '/etc/group'
+]
+SSHD_CONFIG_PATH = '/etc/ssh/sshd_config'
+
+
+def audit_config_files():
+    """
+    For a predefined list of system config files (see AUDITED_CONFIG_FILES)
+    get their last modified time and SHA256 hash.
+    The same info regarding SSHD_CONFIG_PATH is appended (see audit_sshd below),
+    :return: [{'name': ..., 'sha256': ..., 'last_modified': ...}]
+    """
+
+    def digest_sha256(file_path):
+        h = sha256()
+
+        with open(file_path, 'rb') as file:
+            while True:
```

Comment:
Why do you read file and calculate hash in chunks instead of doing this with the full file content?